### DISCUSSION TITLE
[BUG] Icon error in discussion wide-org view

### DISCUSSION THREAD

**Discussion by thanhcsf** [community/community#6005] [Discussions]
I have an error displaying the icon in my organization, see details here:
https://github.com/orgs/mlbvn/discussions

<img width="1208" alt="Screen Shot 2021-10-05 at 22 03 12" src="https://user-images.githubusercontent.com/8481767/136049504-2cf52831-ddcf-4a46-93d6-ce96a23929ac.png">

**Comment by MilosPaunovic** [comment]
Same happens when you're logged in private account and access [https://github.com/discussions](https://github.com/discussions).

![Screenshot 2021-11-15 at 15 54 09](https://user-images.githubusercontent.com/62475782/141802875-780115b9-b741-43d8-8eb0-ec2a2a4ba77a.png)
